### DISCUSSION TITLE
Editor Improved lookups

### DISCUSSION THREAD

**Discussion by Spectreye** [AscensionGameDev/Intersect-Engine#1490] [Ideas]
**Is your feature request related to a problem? Please describe.**
For the moment i'm working streamfully so i can find what i seek. 
But as i'm approaching the 700 items, sometimes i have a hard time to find the items i'm looking for.
And it get worst if i have to delete/remake one cause they get to the bottom of the list and it's really a pain to work with.

Same for the Variables i have actually a hundred of them, but it will grow a lot before the end of the dev (More like a thousand too).

So i was wondering if there could be a way in the editor to get items and variables in Alphabetic order instead of chronogically listed when used in common event / event / Consummable, etc...

Also, a lot of spells and items are getting cutted off by the UI, larger box would be a life savier !

I would happilly help in any way i can (Rebuild Ui or anything else in my skill area) for this to happen, thanks by advance !

**Describe the solution you'd like**

-Give the possibility to have Alphabetic order in listing (Maybe inside the editor options ?)

-Larger Box for Item, Spell, Class, Variable(Player and common).

**Additional context**
![image](https://user-images.githubusercontent.com/68560931/97713991-df759480-1ac0-11eb-9e36-9937f53dc5c9.png)

**Comment by TalikanPME** [comment]
Hello

Is possible have any new of this features ?
I have the same problem...

Thanks

**Comment by jcsnider** [comment]
I've looked into fixing all of this before, but it's more complex than what meets the eye. I would not expect us to overhaul all of this in the short term.

**Comment by WeylonSantana** [comment]
@Cheshire92 If I'm not mistaken, there is a pr of jc that partially solves this one, in terms of organizing the lists.
